packages/editor/src/components/media-placeholder/index.js

```diff
@@ -158,12 +158,46 @@ class MediaPlaceholder extends Component {
 
 		const allowedTypes = this.getAllowedTypes();
 
+		let instructions = labels.instructions || '';
+		let title = labels.title || '';
+		if ( ! instructions || ! title ) {
+			const isVideo = isEmpty( difference( allowedTypes, [ 'video' ] ) );
+			const isImage = isEmpty( difference( allowedTypes, [ 'image' ] ) );
+			const isAudio = isEmpty( difference( allowedTypes, [ 'audio' ] ) );
+			const isImageAndAudio = isEmpty( difference( allowedTypes, [ 'image', 'video' ] ) );
+
+			if ( ! instructions ) {
+				instructions = __( 'Drag a media file, upload a new one or select a file from your library.' );
+
+				if ( isVideo ) {
+					instructions = __( 'Drag a video, upload a new one or select a file from your library.' );
+				} else if ( isImage ) {
+					instructions = __( 'Drag an image, upload a new one or select a file from your library.' );
+				} else if ( isAudio ) {
+					instructions = __( 'Drag an audio, upload a new one or select a file from your library.' );
+				} else if ( isImageAndAudio ) {
```

Comment:
Do you think we should leave this one as a prop (image and video) as it seems too specific to be bundled in the component?

Reply:
Good point, I already left out the title definition for this case.